app/models/account.rb

```diff
@@ -12,4 +12,12 @@ class Account < ApplicationRecord
   def activities
     account_activities
   end
+
+  def as_csv_row
+    [id, balance, currency, registrar]
+  end
+
+  def self.csv_header
+    ['Id', 'Balance', 'Currency', 'Registrar'].freeze
```

Comment:
If you decide to freeze this value and use it as global constant, then you should to move it into the top of file and assigned it like constant `CSV_HEADER = ['Id', 'Balance', 'Currency', 'Registrar'].freeze`, otherwise freeze method not make sense to add. So there are two option: first - move to top and make this value as constant, second - remove freeze method

Reply:
That's true! I will remove the freeze method ;)